docs/docs/dev/architecture/index.md

```diff
@@ -21,6 +21,10 @@ $ plantuml -Tsvg architecture-c4.puml
 
 :::
 
+### Event sourcing
+
+The `hydra-node` is an event sourced application. This means that the main logic is processing _inputs_ (also called commands) and produces _events_. These events are saved and loaded to persist application state across restarts. Also, most events are transformed to _outputs_ and can be observed on the API. See [Event Sourcing](./architecture/event-sourcing) for details.
```

Comment:
Does it make sense to have two exactly the same sentences in two places?

Reply:
Did not want to invent another introduction. Any concrete suggestion what to say instead here?

Reply:
not really...